### ISSUE TITLE
Add-ins won't install

### ISSUE THREAD

**Issue by robster2001** [mozilla/testpilot#843]
- [Windows 10 build 10586.318 ] Operating system
- [Firefox 46.0.1 Release ] Firefox build (Nightly, Developer Edition, Beta, Release)

---

The Activity Stream & Universal Search apps will not install in my copy of Firefox.  I have deleted almost all of my add-ons and disabled my adblocker on firefox.com, but the app page sits at "Enabling..." for hours  (if I let it) without the app loading.   Oddly, Tab Center loaded without any problem.

**Comment by jamiebally** [comment]
This happens to me as well. I am running Windows 10 build 10586.218 and Firefox 46.0.1

**Comment by TheSilverUnicorn** [comment]
I experienced the same issue. It claimed to be "enabling" from 4:37pm to 5:43 (or thereabout) when I finally closed the tab and opted to try out the new Tab Center instead. Tab Center installed in seconds. 

I am using Windows 10 version 1511 OS Build 10586.318 and Firefox 46.0.1. I hope this information is useful.

**Comment by meandavejustice** [comment]
Hello, @robster2001, @jamiebally, @TheSilverUnicorn It appears you are all on the same build of windows and firefox. Would you mind providing a list of addons you have installed? We might be seeing a conflict.

**Comment by meandavejustice** [comment]
I wonder if we can get something like https://github.com/mozilla/testpilot/issues/684 hooked up for the addon as well. Would really help for cases like these.

**Comment by robster2001** [comment]
Enabled: Tab Center, Test Pilot, uBlock Origin (adblocking turned off for firefox.com)

Disabled: Avast Online Security, Avast Safeprice, Blur, Buffer, Download StatusBar, DuckDuckGo Plus, WOT,

**Comment by TheSilverUnicorn** [comment]
Enabled Extensions: Adblock Plus; ChatZilla; Emoji Cheatsheet for GitHub, Basecamp etc.; Kaspersky Protection; Lightbeam; Share Button for Pinterest; Skype; Tab Center; Test Pilot; WOT

Disabled Extenstions: DigitalPersona Extension; Pin It Button; Star Stable Online

Enabled Appearance: Silvers Unicorn 1

Disabled Appearance: crowd of black cats; default theme

Enabled Plugins set to Always Activate: OpenH264 Video Codec provided by Cisco Systems, Inc.; Primetime Content Decryption Module provided by Adobe Systems, Incorporated

Enabled Plugins set to Ask to Activate: Adobe Acrobat; Google Earth Plugin; Google Talk Plugin; Google Talk Plugin Video Renderer; Google Update; iTunes Application Detector; QuickTime Plug-in 7.7.6; Shockwave Flash; Silverlight Plug-In 

Disabled Plugins: Shockwave for Director

Enabled Services: Facebook

**Comment by AustinGuy** [comment]
Extensions enabled: Adblock Plus (disabled for mozilla.com), Element Hiding Helper for Adblock Plus, Facebook Disconnect, InlineDisposition, Kaspersky Protection, LastPass, NetVideoHunter, Tab Mix Plus, Test Pilot, WOT (Web of Trust)

Extensions disabled: none.

It's worth mentioning that although I couldn't enable Activity Stream or Universal Search, I could enable Tab Center. However, Tab Center didn't work even close to what was described. It was barely usable at all and I could only access a few of my tabs. After trying various things, I disabled it again. Even after I did that, my tabs did not fully revert to their previous functioning and behaved oddly. I had to restart FF to get my tabs working as they had before.
